Update Gregtech and SoG

ALongStringOfNumbers opened the pull request:
Updates Gregtech to 1.15.1

Waiting on https://github.com/Shadows-of-Fire/Shadows-of-Greg/pull/154 to be pulled so that an updated version of SoG can be included, minimizing the refactoring of component assembler recipes that I need to do.

ALongStringOfNumbers committed:
Update Gregtech CE from 1.15.0.721 to 1.15.1.735

ALongStringOfNumbers committed:
Update MBTweaker from 1.5.0 to 1.5.1

ALongStringOfNumbers committed:
Add default zoom values

ALongStringOfNumbers committed:
Update Gregtech CE from 1.15.1.735 to 1.16.1.753

ALongStringOfNumbers committed:
Add identifier for The Moon

ALongStringOfNumbers committed:
Update Gregtech CE from 1.16.1.753 to 1.17.0.764

ALongStringOfNumbers committed:
Update Shadows of Greg from 2.19.0 to 2.20.0

ALongStringOfNumbers committed:
Update the Shadows of Greg config file

ALongStringOfNumbers committed:
Fix component recipe errors

Exaxxion commented:
Changes we'll need to make sure the quest book is consistent with:
- [ ] Addition of `dustRegular` oredict (might come up in relation to e.g. Ore Dictionary Filter?)
- [ ] Fluid Regulators
- [ ] Multiblock Info Tabs / part tooltips (not sure, might come up somewhere in the book)
- [ ] Phosphate ore was renamed to Calcium Triphosphate
- [ ] Quantum Chests / Tanks now keep inventory when broken
- [ ] PAs now require Machine Access Interface to hold machinery
- [ ] PAs now have Distinct Item Bus mode

ALongStringOfNumbers committed:
Update Multiblock Tweaker from 1.5.1 to 1.6.0

Exaxxion committed:
Update Fluids and Autocrafting quest description

Exaxxion committed:
Revise Filters quest description

Exaxxion committed:
Update Covers & Macerator quest descriptions

- Mention fluid regulators
- Remove now incorrect information about macerators voiding byproducts
  if full
- also add some text formatting to a few other LV quests

Exaxxion committed:
Tricalcium Phosphate rename, a couple minor misc tweaks

The rename seems to have affected the wrong kind of dust, so I'm opting
to go with what I think the right names will be when that gets fixed.
Might be a little confusing until then, though.

Misc Stuff:
- Some text formatting on quests without it
- Add OR prerequisite for Nether Cake on Atomic Reconstructor
  (Soul Sand is the gate for this quest)

Exaxxion committed:
Update Processing Array quest description and tasks

- Add info about Machine Access Interface, add it as a required item.

Exaxxion committed:
QB: Multiblock Preview tooltips, dustRegular oredict
